src/lib/plugin_apis/s390.api

```diff
@@ -78,3 +78,36 @@ gchar* bd_s390_zfcp_sanitize_wwpn_input (const gchar *wwpn, GError **error);
  * Returns: (transfer full): a synthesized zFCP LUN
  */
 gchar* bd_s390_zfcp_sanitize_lun_input (const gchar *lun, GError **error);
+
+/**
+ * bd_s390_zfcp_online:
+ * @devno a zFCP device number
+ * @wwpn a zFCP WWPN identifier
+ * @lun a zFCP LUN identifier
+ * @error: (out): place to store error (if any)
+ *
+ * Returns: whether a zFCP device was successfully switched online
+ */
+gboolean bd_s390_zfcp_online (const gchar *devno, const gchar *wwpn, const gchar *lun, GError **error);
+
+/*
+ * bd_s390_zfcp_scsi_online:
+ * @devno a zFCP device number
+ * @wwpn a zFCP WWPN identifier
+ * @lun a zFCP LUN identifier
+ * @error: (out): place to store error (if any)
+ *
+ * Returns: whether a LUNs were successfully removed from associated WWPN
```

Comment:
Extra "a"? Or maybe a wrong sentence? I don't see how such parameters could result into multiple LUNs being taken offline. It would be nice to make this description a bit more verbose so that it makes more sense in relation to the name of the function.